libcni/api.go

```diff
@@ -39,6 +41,7 @@ type RuntimeConf struct {
 	ContainerID string
 	NetNS       string
 	IfName      string
+	Timeout     time.Duration
```

Comment:
Instead of having a Timeout here, just have callers of libcni pass their own Context (or nil) into the calls like AddNetworkList, DelNetworkList, etc.